mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala

```diff
@@ -152,7 +152,9 @@ final class ActorMqttClientSession(settings: MqttSessionSettings)(implicit mat:
 
   private val pingReqBytes = PingReq.encode(ByteString.newBuilder).result()
 
-  override def commandFlow[A]: CommandFlow[A] =
+  override def commandFlow[A](connectionId: ByteString): CommandFlow[A] = {
+    val reply = Promise[Source[ClientConnector.ForwardConnectCommand, NotUsed]]
+
```

Comment:
I'm unsure why this `reply` declaration has been moved...

Reply:
That said, moving it is benign.

Reply:
Ah, this was a leftover as I was iterating on things. Thanks for noticing. I shall fix in a bit.

Reply:
This is fixed.